homeassistant/components/firmata/config_flow.py

```diff
@@ -0,0 +1,39 @@
+"""Config flow to configure firmata component."""
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_NAME
+from homeassistant.core import callback
+
+from .const import CONF_SERIAL_PORT, DOMAIN
```

Comment:
Disable the pylint warning for unused import of `DOMAIN`.

Reply:
I'm confused. It is being used?

Reply:
There's a pylint bug for the class keyword argument analysis.

Reply:
Do you know what the exclusion code is? I cannot replicate this bug so I am unsure what it is.

Reply:
Aha! https://github.com/home-assistant/core/pull/33613/files/961e0d5729fc499aa7700c0c2099c02518e0574f#r430205066